src/mumble/AudioWizard.cpp

```diff
@@ -306,7 +306,7 @@ void AudioWizard::showPage(int pageid) {
 	AudioOutputPtr ao = Global::get().ao;
 	if (ao)
 		ao->wipe();
-	aosSource = nullptr;
+	m_chord = AudioOutputToken::None();
```

Comment:
```suggestion
	m_chord = {};
```
Use new default constructor. Alternatively, use `reset` function, if you choose to implement it